### DISCUSSION TITLE
power meter is always 1kWh behind the real value

### DISCUSSION THREAD

**Discussion by Eisbahn** [jomjol/AI-on-the-edge-device#2673] [General]
Hi,

everything is working and I get perfect detection of my power meter since months.
Just one very minor thing I just realized: the values are always 1 kWh  behind the real ones, e.g. a value of 28496(kWh) is reported but the real value is 28497,8kWh. I'm using the latest official version v15.3.0 (Commit: 3fbff0a) with ROI model dig-class100-0160_s2_q.tflite.
Please treat this is minor issue and I don't care that much, but if I could get rid of it, any help is welcome.

![Data_Recognition](https://github.com/jomjol/AI-on-the-edge-device/assets/336105/a3b6cb68-a0d8-463b-a807-439d6c3062c7)

**Comment by Eisbahn** [comment]
and no, also the actual one shows the same "-1kWh, the "seven" last digit was present for longer time. I'm not cooking/drawing much power at the moment :-)

**Comment by Eisbahn** [comment]
![overview](https://github.com/jomjol/AI-on-the-edge-device/assets/336105/552e71d0-162a-4141-aa31-538dd0bc79a7)

**Comment by friedpa** [comment]
I remember something with "Extended Resolution", as far as I think it must be set to false in your case, but I am not sure anymore. My meter works like yours and this parameter is also set to false.

**Reply by Eisbahn** [reply]
Hi,
it was already on false.

Other settings are:
prev. value = true
allow neg. rates = false
dec shift = 0
max rate value = 5
max rate change = absolute change
ext. resolution = false
ignore leading NaN = false
